tests/v2/test_waku_discv5.nim

```diff
@@ -119,3 +119,65 @@ procSuite "Waku Discovery v5":
     #   (await completionFut.withTimeout(6.seconds)) == true
 
     await allFutures([node1.stop(), node2.stop(), node3.stop()])
+
+  asyncTest "Custom multiaddresses are advertised correctly":
+    let
+      bindIp = ValidIpAddress.init("0.0.0.0")
+      extIp = ValidIpAddress.init("127.0.0.1")
+      expectedMultiAddr = MultiAddress.init("/ip4/200.200.200.200/tcp/9000/wss").tryGet()
+
+      flags = initWakuFlags(lightpush = false,
+                            filter = false,
+                            store = false,
+                            relay = true)
+
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      nodeTcpPort1 = Port(9010)
+      nodeUdpPort1 = Port(9012)
+      node1 = WakuNode.new(nodeKey = nodeKey1,
+                           bindIp = bindIp,
+                           extIp = some(extIp),
+                           extPort = some(nodeTcpPort1),
+                           bindPort = nodeTcpPort1,
+                           discv5Enabled = true,
+                           discv5UdpPort = some(nodeUdpPort1),
+                           extMultiAddrs = @[expectedMultiAddr],
+                           discv5TableIpLimit = some(10'u),
+                           discv5BucketIpLimit = some(10'u),
+                           discv5BitsPerHop = some(10),
+                           wakuFlags = some(flags))
+
+
+      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
```

Comment:
minor suggestion, direction use `generateKey()`
`WakuNode.new(nodeKey=generateKey()))`

(see test_helpers.nim)

Reply:
Addressed in [185a10a](https://github.com/waku-org/nwaku/pull/1512/commits/185a10ad6637c8c13f2e58c5d7305d290c6a3982), thanks!